# scalings
linear_scale: 1.0 # [units]
angular_scale: 1.0 # [units]
max_linear: 0.1 # [m/s]
max_angular: 1.0 # [r/s]

# timeouts
device_timeout: 30.0 # [s]
central_timeout: 1.0 # [s]

# ROS parameters
queue_size: 10 # [units]

# tunings
device_strings: ["joystick", "turtlebot", "alphabot", "gopigo", "quad"]

#             linear scale  | angular scale  | linear bound  | angular bound
devices:
  joystick:  [1.00,           1.00,            1.00,           1.00]
  turtlebot: [1.00,           1.00,            0.60,           0.99]
  alphabot:  [0.50,           1.00,            0.30,           0.99]
  gopigo:    [1.00,           1.00,            0.60,           0.99]
  quad:      [1.00,           1.00,            0.60,           0.99]
ui:          [0.05,           0.33,            1.00,           1.00]
